<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newtvshow</class>
 <widget class="QMainWindow" name="newtvshow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Completed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tvRating">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>TV-Y, TV-Y7, TV-G, TV-PG, TV-14, TV-MA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Location/Platform</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Episode #</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Season #</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="episode_Number">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ex - Type 1 for episode 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEpisode">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Episode</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateAdded">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="season_Number">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ex - Type 1 for season 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IMDB #</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tvLocation">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Theater, Home, streaming, physcial media</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date Added</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IMDB_Number">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ex - /tt0468569/ after title in IMDB URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchTV">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tvCompletion">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Watched in full?</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tvinfo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="airDate">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Air Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ep Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="episodeDescription">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Episode Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="episodeTitle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="showTitle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="genre">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ex - Crime, Drama, Sci-Fi, Comedy, Childrens</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="episodeDuration">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ex - 65 Minutes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Episode Rating</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="episodeRating">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ex - IMDB Episode Rating</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showDescpription">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>715</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Instructions</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>750</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Input IMDB ID # for show being added</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>770</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>770</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Input Season # and Episode # for episode being added</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>790</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3. Hit 'Search' button above</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>810</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4. Application will retreive info for episode being added</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>830</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5. Fill out rest of form using info returned based on search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>850</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6. Click 'Add Episode' to add to database</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
